public/app/percona/dbaas/components/DBCluster/EditDBClusterPage/DBClusterBasicOptions/Restore/Restore.test.tsx

```diff
@@ -0,0 +1,94 @@
+import { screen, render, waitFor, fireEvent } from '@testing-library/react';
+import React from 'react';
+import { Form } from 'react-final-form';
+import { Provider } from 'react-redux';
+
+import { configureStore } from '../../../../../../../store/configureStore';
+import { StoreState } from '../../../../../../../types';
+
+import { Restore } from './Restore';
+
+jest.mock('app/percona/dbaas/components/Kubernetes/Kubernetes.service');
+jest.mock('app/percona/backup/components/StorageLocations/StorageLocations.service');
+
+const store = configureStore({
+  percona: {
+    user: { isAuthorized: true },
+    kubernetes: {
+      loading: false,
+    },
+  },
+} as StoreState);
+describe('DBaaS DBCluster creation Restore section ::', () => {
+  it('renders items correctly, shows fields on switch on', async () => {
+    await waitFor(() =>
+      render(
+        <Provider store={store}>
+          <Form onSubmit={jest.fn()} render={({ form }) => <Restore form={form} />} />
+        </Provider>
+      )
+    );
+    expect(screen.getByTestId('toggle-scheduled-restore')).toBeInTheDocument();
+
+    const checkbox = screen.getByTestId('toggle-scheduled-restore');
+
+    const consoleWarnMock = jest.spyOn(console, 'warn').mockImplementation();
```

Comment:
Why is there a check for a deprecation warning?

Reply:
Oh! It was a big question for me!
I used standart grafana `switch` component
![image](https://user-images.githubusercontent.com/90199600/216916550-8cdee674-32bf-4238-a0e4-f54a9df19553.png)
but as I understand grafana `switch` uses the deprecated `checked` prop 
Screen from packages/grafana-ui/src/components/Switch/Switch.tsx
![image](https://user-images.githubusercontent.com/90199600/216918075-01456164-c733-4f16-8047-718c9fb450ab.png)

when I tryied to use standart way to test this switch like
` await waitFor(() => fireEvent.change(checkbox, { target: { value: true } }));`
it didn't work, fireEvent didn't change anything in my tests
also when I tried to use "checked" prop in target object

so I tryied to use click, and click worked, but tests return to me a warning
I disabled warning just for this click, becouse I haven't found any other way to enable this component in tests

Reply:
I also checked all places where this component is used (for example, in current backups (not dbaas backups)) and there are no tests that would work with <Switch/>

Reply:
If you have any suggestions on how to switch the `<Switch/>` in tests to true, I will be happy to try them,
but the standard way, as I wrote above, through the use of `fireEvent.change` does not work with either `value` or `checked`

Reply:
@solovevayaroslavna 
It looks like the input from react-final-form is adding the checked prop:

<img width="799" alt="image" src="https://user-images.githubusercontent.com/32638572/216923251-d949a96c-c39d-4b10-bfa8-09ba18ef5654.png">

When I've set it to undefined there was no warning, the "renders items correctly, shows fields on switch on", altought the "shows secrets field if kubernetesCluster name exists in form" failed.

Reply:
Thanks a lot! It's interesting. I fixed myTests with this prop and delete deprication warning